index.js

```diff
@@ -55,6 +56,7 @@ module.exports = {
       }
 
       var linted = trees.map(function(tree) {
+        let filteredTreeToBeLinted = new Funnel(tree, { exclude: '**/*.js' });
         return new ScssLinter(mergeTrees([tree]), this.scssLintOptions);
```

Comment:
No problem. I am working through this PR now. Upgrading ember-cli and all that. Should get it pushed soon. I'll include this change